src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-share-form/smb-share-form.component.html

```diff
@@ -206,6 +212,7 @@
         (submitActionEvent)="submitAction()"
         [form]="smbShareForm"
         [submitText]="(action | titlecase) + ' ' + (resource | upperFirst)"
+        [disabled]="smbShareForm.pending || smbShareForm.invalid"
```

Comment:
@Pegonzal: can we make this generic across all forms? We are already passing the `form` to the component, so maybe we can do the same thing in the actual component which gets applied to all the forms?

If you feel that as out of scope maybe a small separate PR to take care of it? wdyt?

Reply:
Added it to all forms, the only thing it worries me is the testing side, as I did not get to test all the forms

Reply:
if the basic e2e is passing, we might be good.